src/main/org/pantsbuild/jarjar/util/JarTransformerChain.java

```diff
@@ -17,12 +17,14 @@
 package org.pantsbuild.jarjar.util;
 
 import org.objectweb.asm.ClassVisitor;
+import org.pantsbuild.jarjar.misplaced.MisplacedClassStrategy;
 
 public class JarTransformerChain extends JarTransformer
 {
     private final RemappingClassTransformer[] chain;
     
-    public JarTransformerChain(RemappingClassTransformer[] chain) {
+    public JarTransformerChain(MisplacedClassStrategy strategy, RemappingClassTransformer[] chain) {
```

Comment:
Seeing this makes me think of how much burden we are putting on the API to plumb this option through to support what is probably a feature that no one wants.  Still thinking about this...